website/content/docs/job-specification/lifecycle.mdx

```diff
@@ -26,7 +26,7 @@ tasks or "sidecar" tasks that are expected to run for the duration of the main t
 The absence of the sidecar flag indicates that the task is ephemeral
 and should not be restarted if it completes successfully.
 
-Learn more about Nomad's task dependencies on the [HashiCorp Learn website][learn-taskdeps].
+Learn more about Nomad's task dependencies [here][learn-taskdeps].
```

Comment:
```suggestion
Learn more about [Nomad's task dependencies][learn-taskdeps].
```
SEO benefits if links have meaningful content (instead of words like "here").